### DISCUSSION TITLE
New release broke functionality?

### DISCUSSION THREAD

**Discussion by master-of-none** [zed-industries/zed#7030] [Feature Requests]
Did the new release break many functionalities of Zed?
I cannot change any settings and settings.json takes longer to save and the change isn't reflected anywhere.

**Comment by mrnugget** [comment]
Hey! What version of Zed are you on now? I'm on `0.119.21` and I can change `settings.json` and the changes are reflected. I changed the UI. 

Do you have anything suspicious in `~/Library/Logs/Zed/Zed.log`?

**Reply by SomeoneToIgnore** [reply]
Another possibility is that the settings.json contents got malformed, so Zed fails to parse it and fails to defaults.

**Reply by master-of-none** [reply]
I was in the new version. As @SomeoneToIgnore said the json file got malformed. Apparently there were comments in the beginning of the file and when I save it extra space was getting added and json was considering it as invalid character. Anyway I removed the comments and it's fixed now. 
Thank you 😄
